exhibitor-standalone/src/main/java/com/netflix/exhibitor/servlet/ExhibitorServletFilter.java

```diff
@@ -36,7 +36,7 @@ public void init(FilterConfig filterConfig) throws ServletException
     public void doFilter(ServletRequest request, ServletResponse response, FilterChain chain) throws IOException, ServletException
     {
         HttpServletResponse httpServletResponse = (HttpServletResponse)response;
-        httpServletResponse.sendRedirect("/exhibitor/v1/ui/index.html");
+        httpServletResponse.sendRedirect("./exhibitor/v1/ui/index.html");
```

Comment:
I am currently travelling,  I will test and send another pull request,
Thank you
On Mar 4, 2015 11:57 AM, "Jordan Zimmerman" <notifications@github.com>
wrote:

> In
> exhibitor-standalone/src/main/java/com/netflix/exhibitor/servlet/ExhibitorServletFilter.java
> <https://github.com/Netflix/exhibitor/pull/234#discussion_r25790927>:
>
> > @@ -36,7 +36,7 @@ public void init(FilterConfig filterConfig) throws ServletException
> >      public void doFilter(ServletRequest request, ServletResponse response, FilterChain chain) throws IOException, ServletException
> >      {
> >          HttpServletResponse httpServletResponse = (HttpServletResponse)response;
> > -        httpServletResponse.sendRedirect("/exhibitor/v1/ui/index.html");
> > +        httpServletResponse.sendRedirect("./exhibitor/v1/ui/index.html");
>
> Could this also be done as:
>
> httpServletResponse.sendRedirect("exhibitor/v1/ui/index.html");
>
> Seems cleaner to me
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/Netflix/exhibitor/pull/234/files#r25790927>.
>